ts/features/pushNotifications/utils/configurePushNotification.ts

```diff
@@ -41,7 +51,7 @@ export const configurePushNotifications = (store: Store) => {
       importance: 4,
       vibrate: true
     },
-    constNull
+    _created => null
```

Comment:
why remove the `constNull`?

Reply:
I did it because it was hiding the "created" parameter to the reader. In this way, it is immediately clear (to the reader) that there is a way to know if the channel has been properly created, without having to go to the documentation